### ISSUE TITLE
How to load your pretrained DiT model in HuggingFace for LoRA finetune?

### ISSUE THREAD

**Issue by TechnetiumMan** [stepfun-ai/Step1X-3D#30]
I want to load your HuggingFace DiT weights, like your inference.py code: 
`pipeline = Step1X3DGeometryPipeline.from_pretrained(
        "stepfun-ai/Step1X-3D", subfolder='Step1X-3D-Geometry-Label-1300m'
    ).to("cuda")`
But I don't know how to load it in LoRA training code. 

I tried to use your LoRA finetune script:
`CUDA_VISIBLE_DEVICES=0 python train.py --config $config --train --gpu 0 system.use_lora=True`
but it seems to init the DiT model without loading your pretrain DiT weights, and only set LoRA params to trainable. (of course, it didn't work)

Can you tell me how to do? Thanks.

我想要加载你们HuggingFace上的DiT预训练模型，就像inference.py中做的那样：
`pipeline = Step1X3DGeometryPipeline.from_pretrained(
        "stepfun-ai/Step1X-3D", subfolder='Step1X-3D-Geometry-Label-1300m'
    ).to("cuda")`
但我不知道如何在LoRA训练代码中加载DiTch预训练模型。

我尝试了项目ReadMe中的LoRA微调训练：
`CUDA_VISIBLE_DEVICES=0 python train.py --config $config --train --gpu 0 system.use_lora=True`
但是似乎没有加载任何DiT模块权重，只是将其初始化后锁定，只允许LoRA参数更新。那当然什么都训不出来。

请问应该如何去做？谢谢！
